certify-postgres-landregistry.properties

```diff
@@ -25,6 +36,16 @@ mosip.certify.data-provider-plugin.postgres.scope-query-mapping={\
   }
 
 
+## ------------------------------------------- Cache specific usecase properties ------------------------------------------------------------
+# Velocity Template caching for performance of VCI -- required when Certify is deployed in plugin-mode=DataProvider
+mosip.certify.cache.names=templatecache
+spring.cache.cache-names=${mosip.certify.cache.names}
+# going with a modest size considering no multi-tenancy
+mosip.certify.cache.size={templatecache': 20}
+# cache duration set to half-day
+mosip.certify.templatecache-expire-seconds=43200
+mosip.certify.cache.expire-in-seconds={'templatecache': ${mosip.certify.templatecache-expire-seconds}}
```

Comment:
Can we not keep the cache section in the certify-default? Normally usecase specific, they will not change these much

Reply:
The `templatecache` is required only by `DataProvider` plugin and not the other plugin implementations such as `mosipid`, `insurance`, `mock`, etc.

Reply:
that is ok, this config ideally will not create any issues right ?